soco/core.py

```diff
@@ -2251,8 +2264,26 @@ def get_battery_info(self, timeout=3.0):
     "SHUFFLE_REPEAT_ONE",
     "REPEAT_ONE",
 )
+
+# URI prefixes for music sources
+SOURCES = {
+    r"^$": "NONE",
+    r"^x-file-cifs:": "LIBRARY",
+    r"^x-rincon-mp3radio:": "RADIO",
+    r"^x-sonosapi-stream:": "RADIO",
+    r"^x-sonosapi-radio:": "RADIO",
+    r"^x-sonosapi-hls:": "RADIO",
+    r"^aac:": "RADIO",
+    r"^hls-radio:": "RADIO",
+    r"^https?:": "WEB_FILE",
+    r"^x-rincon-stream:": "LINE_IN",
+    r"^x-sonos-htastream:": "TV",
+    r"^x-sonos-vli:.*,airplay:": "AIRPLAY",
```

Comment:
I think it would be good to define constants like `MUSIC_SOURCE_RADIO = "RADIO"` that clients could import to avoid typos.

Reply:
Done.